### ISSUE TITLE
Add macOS support

### ISSUE THREAD

**Issue by robinwassen** [robinwassen/electron-wallpaper#1]
Do a best effort implementation even if it might not be possible to create a solution that is as good as it is on Windows.

**Comment by drozdzynski** [comment]
Hello,
are there any progress with the MacOS version?

**Comment by nickforddev** [comment]
+1  any chance of this happening? i am new to electron and trying to create an electron version of http://tracesof.net/uebersicht/

**Comment by hendrikreimers** [comment]
+1
For macOS it's easy i think and no need of native module (if i remember correctly).
But i wanna do the same for windows and mac too in future!

**Comment by nickforddev** [comment]
Yes actually I just found in the docs that you can specify that `type: 'desktop'` for Mac, it works perfectly!
